### DISCUSSION TITLE
GLD_TEXTURE_INDEX_2D is unloadable

### DISCUSSION THREAD

**Discussion by multitudes** [codam-coding-college/MLX42#124] [Q&A] [answered]
Hi!
I get this log error message. I am not sure if it is a bug so I will put it here:
> "UNSUPPORTED (log once): POSSIBLE ISSUE: unit 2 GLD_TEXTURE_INDEX_2D is unloadable and bound to sampler type (Float) - using zero texture because texture unloadable"

I am on an M1 MacBook Pro with the last version of Sonoma 14.2.1. I am using the MLX lib for the first time on the mac. On Ubuntu 22 I get no such message and the program (42-Fractol) works fine.
Thank you for suggestions

**Comment by W2Wizard** [comment, marked as answer]
Duplicate #29
